### PR TITLE
Add database objects definition parser

### PR DESCRIPTION
**Pull request by epicallan**
Extraction of #189

* [x] Depends on #194

### CODE REVIEW

File: pgt/src/PGT/Output/Text.hs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module PGT.Output.Text
+  ( Render(..)
+  )
+where
+
+class Render a where
```
Comment (epicallan) [inline pgt/src/PGT/Output/Text.hs:21]: TODO: move to own module

File: pgt/src/PGT/Output/RowCount.hs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+module PGT.Output.RowCount
```
Comment (epicallan) [inline pgt/src/PGT/Output/RowCount.hs:1]: TODO: move to separate PR.

**Review by dkubb — APPROVED** [review]
